.github/release-drafter.yml

```diff
@@ -9,11 +9,15 @@ categories:
       - 'release/breaking-change'
 change-template: '- $TITLE @$AUTHOR (#$NUMBER)'
 template: |
+  This is a draft for the next release.
+
+  Please do not edit this file directly, use it as a template in the next release.
+
   ## Changes
 
   $CHANGES
 
   ## Docker images
 
   - `docker pull kudobuilder/controller:latest`
-  - `docker pull kudobuilder/controller:$NEXT_PATCH_VERSION`
+  - `docker pull kudobuilder/controller:EDITME`
```

Comment:
lets pull the whole docker section out (as in https://github.com/kudobuilder/kudo/pull/1379)
This is a template to augment the goreleaser release data.   goreleaser includes this data and I would propose is the release authority on data.   What we need from this process is release highlights and breaking changes... everything else looks like it will be lost.